stubs/dateparser/dateparser/date.pyi

```diff
@@ -88,10 +88,10 @@ class DateDataParser:
     locale_loader: ClassVar[LocaleDataLoader | None]
     try_previous_locales: bool
     use_given_order: bool
-    languages: Any
-    locales: Any
-    region: Any
-    previous_locales: Any
+    languages: list[str] | None
+    locales: list[str] | tuple[str] | set[str] | None
+    region: str
+    previous_locales: dict[Locale, None]
```

Comment:
This is a `collections.OrderedDict`, according to the implementation.

Reply:
Good catch 👍 
Updated.